variants/STM32G0xx/G031C(4-6-8)(T-U)_G041C(6-8)(T-U)/generic_clock.c

```diff
@@ -24,8 +24,42 @@
   */
 WEAK void SystemClock_Config(void)
 {
-  /* SystemClock_Config can be generated by STM32CubeMX */
-#warning "SystemClock_Config() is empty. Default clock at reset is used."
+  RCC_OscInitTypeDef RCC_OscInitStruct = {0};
```

Comment:
Some remaining '0'

Reply:
How is notepad ++ missing those. Will find a RegEx to scan for them
Thank you for spotting these, sorry for missing them

Reply:
Fixed